remind.py

```diff
@@ -347,20 +347,28 @@ def _gen_msg(vevent, label):
         if hasattr(vevent, 'location') and vevent.location.value:
             msg.append('at %s' % vevent.location.value.strip())
 
-        if hasattr(vevent, 'description') and vevent.description.value:
+        has_desc = hasattr(vevent, 'description') and vevent.description.value
+
+        if tail or has_desc:
             rem.append('%%"%s%%"' % ' '.join(msg))
-            rem.append(vevent.description.value.strip())
         else:
             rem.append(' '.join(msg))
+        if tail:
+            rem.append(tail)
+
+        out = ' '.join(rem)
+        if has_desc:
+            out += sep + vevent.description.value.strip()
```

Comment:
The reason I've done it this way is that I don't necessarily want a space there. The situation I want is where an entry with summary ‘Foo’ and description ‘Bar’ is transformed not to `%"Foo%" Bar`, but rather to `%"Foo%"%_Bar`.
With the append functionality, this would instead become `%"Foo%" %_ Bar`. Then again, this may not be a big enough problem to justify the slight mess it makes in the code.